libnopegl/src/node_rtt.c

```diff
@@ -143,7 +153,7 @@ static int rtt_init(struct ngl_node *node)
             return NGL_ERROR_INVALID_ARG;
         }
 
-        const struct texture_priv *texture_priv = info.texture_priv;
+        struct texture_priv *texture_priv = info.texture_priv;
```

Comment:
Why do you need to change constness in this commit?

Reply:
Because we need to write to the underlying texture params struct later on: `params->usage |= ...`